wp-content/themes/access/views/partials/nav.twig

```diff
@@ -1,3 +1,8 @@
-<a href="{{ url_base }}/eligibility#step-1" {% if this.style %}class="{{ this.style }}"{% endif %} data-js="track" data-track-key="Main Nav" data-track-data='[{"action": "eligibility#step-1"}]'>{{ __('Am I eligible?', 'accessnyctheme') }}</a>
-<a href="{{ url_base }}/programs" {% if this.style %}class="{{ this.style }}"{% endif %} data-js="track" data-track-key="Main Nav" data-track-data='[{"action": "programs"}]'>{{ __('Programs', 'accessnyctheme') }}</a>
-<a href="{{ url_base }}/locations" {% if this.style %}class="{{ this.style }}"{% endif %} data-js="track" data-track-key="Main Nav" data-track-data='[{"action": "locations"}]'>{{ __('Find help', 'accessnyctheme') }}</a>
+{% for m in header_menu.get_items %}
```

Comment:
Copied from https://github.com/CityOfNewYork/ACCESS-NYC/blob/f04b3256e3d188a84a7e5286e04651ecb46519bb/wp-content/themes/access/views/objects/footer.twig#L6

I removed the HTML tags that appeared to be for tracking (`data-js`, `data-track-key`, and `data-track-data` because it seems as though these aren't actually used in our analytics products.